api/v1beta1/etcdadmconfig_types.go

```diff
@@ -197,6 +200,11 @@ type RegistryMirrorConfiguration struct {
 	CACert string `json:"caCert,omitempty"`
 }
 
+// BottlerocketKernelSettings holds the kernel settings for bottlerocket nodes.
+type BottlerocketKernelSettings struct {
+	SysctlSettings map[string]string `json:"sysctlSettings,omitempty"`
+}
```

Comment:
Can you reuse the struct introduced [here](https://github.com/abhay-krishna/cluster-api/pull/17) instead of re-defining it?